sub-packages/bionemo-evo2/README.md

```diff
@@ -265,6 +265,12 @@ predict_evo2 \
 
 An example of using `predict_evo2` for variant effect prediction can be found in our [Evo 2 Zeroshot BRCA1 Notebook](https://docs.nvidia.com/bionemo-framework/latest/user-guide/examples/bionemo-evo2/evo2_zeroshot_brca). This notebook demonstrates how to use Evo2 to predict whether single nucleotide variants (SNVs) in the BRCA1 gene are likely to be harmful to protein function and potentially increase cancer risk, by comparing the model's log probability scores between the reference and variant sequences.
 
+## Context Extension
+
+Evo2 supports extended context lengths beyond its pretraining limit.
```

Comment:
This seems like it could be interpreted in different ways from what we intend. Maybe something like the following (feel free to change):

Evo2 supports continuing training with longer context lengths beyond those used to train a prior checkpoint. For example when training the original Evo2 model, the first phase of training was performed at 8192 context length while the next phase continued training at 1m context length, but starting from the prior 8192 context length checkpoint. We call this process context extension.